### PR TITLE
Added constant "\PDO::MYSQL_ATTR_SSL_VERIFY_SERVER_CERT"

### PR DESCRIPTION
**Pull request by Geolim4**
- As per https://bugs.php.net/bug.php?id=71003
- Implemented at http://git.php.net/?p=php-src.git;a=blobdiff;f=ext/pdo_mysql/pdo_mysql.c;h=99fb6d3b77154c2f9928dd90e4f3c4b4f76d9611;hp=32b85ba54736e4f5bc79ceb5a486242d72da511a;hb=247ce052cd0fc7d0d8ea1a0e7ea2075e9601766a;hpb=b224e7426713befd7449117afd68355231f9077e
- As of 7.0.18 and 7.1.4 (see http://php.net/ChangeLog-7.php)
- PR at php/php-src#1972

### CODE REVIEW

File: PDO/PDO.php [hunk 1]
```diff
@@ -710,6 +710,18 @@ class PDO  {
...
 	const MYSQL_ATTR_MULTI_STATEMENTS = 1015;
 
+	/**
+	 * <p>
+	 * Disables SSL peer verification when set to FALSE.
+	 * </p>
+	 * <p>
+	 * Note, this constant is not yet documented on the php.net documentation.
+	 * </p>
+	 * @since 7.0.18 and 7.1.4
```
Comment (b1rdex) [inline PDO/PDO.php]: This should be splitted to two `@since` tags

**Comment by b1rdex** [comment]
Is there any request to add this to php.net documentation?

File: PDO/PDO.php (hunk 1 above)
Comment (Geolim4) [reply PDO/PDO.php]: I can split them but what about this then ?
https://github.com/JetBrains/phpstorm-stubs/blob/master/PDO/PDO.php#L708

**Comment by Geolim4** [comment]
I'm not sure, in my case I did no request at all.

File: PDO/PDO.php (hunk 1 above)
Comment (b1rdex) [reply PDO/PDO.php]: dunno 🤷‍♂️  If this ever become parsed, then it's better to have them splitted

Comment (b1rdex) [reply PDO/PDO.php]: Also, take a look at `@since` definition at http://docs.phpdoc.org/references/phpdoc/tags/since.html

Comment (Geolim4) [reply PDO/PDO.php]: Okay, I'll fix mine + another PR to fix others, @andrey-sokolov what's your mind on this ?

Comment (andrey-sokolov) [reply PDO/PDO.php]: For new contributions it's better ti split such tags

**Comment by Geolim4** [comment]
I can fix it tonight then, thanks :)

Le mar. 24 avr. 2018 à 15:04, andrey-sokolov <notifications@github.com> a
écrit :

File: PDO/PDO.php [hunk 2]
```diff
@@ -710,6 +710,18 @@ class PDO  {
 	 */
 	const MYSQL_ATTR_MULTI_STATEMENTS = 1015;
 
+	/**
+	 * <p>
+	 * Disables SSL peer verification when set to FALSE.
+	 * </p>
+	 * <p>
+	 * Note, this constant is not yet documented on the php.net documentation.
```
Comment (andrey-sokolov) [inline PDO/PDO.php]: I think, this note is redundant and it's better to remove it

Comment (Geolim4) [reply PDO/PDO.php]: Okay, i thought that it was a good idea for those that have to maintain the project.

Comment (Geolim4) [reply PDO/PDO.php]: @andrey-sokolov It's fixed !

Comment (Geolim4) [reply PDO/PDO.php]: @andrey-sokolov up :)

Comment (Geolim4) [reply PDO/PDO.php]: @andrey-sokolov Up !

**Pull request merged**